packages/@uppy/companion/src/server/helpers/request.js

```diff
@@ -85,9 +85,9 @@ const getProtectedHttpAgent = ({ protocol, blockLocalIPs }) => {
 
 module.exports.getProtectedHttpAgent = getProtectedHttpAgent
 
-async function getProtectedGot ({ blockLocalIPs }) {
-  const HttpAgent = getProtectedHttpAgent({ protocol: 'http', blockLocalIPs })
-  const HttpsAgent = getProtectedHttpAgent({ protocol: 'https', blockLocalIPs })
+async function getProtectedGot ({ allowLocalIPs }) {
```

Comment:
should this have a `= false` default too?

Reply:
`undefined` has the same function as `false`, so it shouldn't be necessary I think. but feel free to change it if you think i tshould.